tffm/core.py

```diff
@@ -91,22 +97,32 @@ class TFFMCore():
     ----------
     Steffen Rendle, Factorization Machines
         http://www.csie.ntu.edu.tw/~b97053/paper/Rendle2010FM.pdf
+
     """
     def __init__(self, order=2, rank=2, input_type='dense', loss_function=utils.loss_logistic, 
-                optimizer=tf.train.AdamOptimizer(learning_rate=0.01), reg=0, init_std=0.01, 
-                use_diag=False, reweight_reg=False, seed=None):
+                optimizer=tf.train.AdamOptimizer(learning_rate=0.01), reg=0,
+                init_std=0.01, class_weight=None, use_diag=False, reweight_reg=False,
```

Comment:
IMHO, class_weight should be parameter of `TFFMClassifier.fit` method, not constructor of `Base`. (and can be changed during training without creating new class instance)

Reply:
Agreed.